tools/packaging/kernel/README.md

```diff
@@ -49,14 +54,16 @@ Options:
 
 Example:
 ```
-$ ./build-kernel.sh -v 4.19.86 -g nvidia -f -d setup
+$ ./build-kernel.sh -v 5.10.25 -g nvidia -f -d setup
 ```
 > **Note**
-> - `-v 4.19.86`: Specify the guest kernel version.
+> - `-v 5.10.25`: Specify the guest kernel version. 
```

Comment:
Also nit: whitespace

Reply:
I don't mind rebasing in fixups, but I gather that will re-trigger the need for all tests. It looks like that's kind of expensive (maintainers rerunning false failures, etc.). Let me know community preference on that.